app/controllers/users_controller.rb

```diff
@@ -6,4 +6,23 @@ def index
   def show
     @user = User.find(params[:id])
   end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to @user, notice: 'User was successfully created.'
```

Comment:
以下のように日本語にした方がいいかと思う！
```suggestion
      redirect_to @user, notice: 'アカウントの作成に成功しました'
```

Reply:
[9bf91a7](https://github.com/ok-os-job-change-team/tetsuya-twitter-clone-bootcamp/pull/34/commits/9bf91a769c2d8d9a714d2ba68e93ccf672aa9352) で変更しました。